src/components/Issuance/EmailIssuance.tsx

```diff
@@ -102,28 +131,76 @@ const EmailIssuance = () => {
 
 	const confirmOOBCredentialIssuance = async () => {
 		setIssueLoader(true);
+		
 		const existingData = userData;
+		const organizationDid = await getFromLocalStorage(storageKeys.ORG_DID);
+		
+	let transformedData: any = { credentialOffer: [] };
+	
+if (existingData && existingData.formData) {
+    if (schemaType === SchemaTypes.schema_INDY) {
+        existingData?.formData.forEach((entry: { email: any; attributes: any[] }) => {
+            const transformedEntry = { emailId: entry.email, attributes: [] };
+            entry.attributes.forEach((attribute) => {
+                const transformedAttribute = {
+                    value: String(attribute.value || ''),
+                    name: attribute.name || '',
+                    isRequired: attribute.isRequired,
+                };
+                transformedEntry.attributes.push(transformedAttribute);
+            });
+            transformedData.credentialOffer.push(transformedEntry);
+        });
+        transformedData.credentialDefinitionId = credDefId;
+		
+    } else if (schemaType=== SchemaTypes.schema_W3C) {
+        existingData.formData.forEach((entry: { email: any; credentialData: IEmailCredentialData; attributes:IIssueAttributes[] }) => {
+			const credentialOffer = {
+				emailId: entry.email,
+                credential: {
+					"@context": [
+						"https://www.w3.org/2018/credentials/v1",
```

Comment:
We can move these hardcoded values to one common file and use those variables when forming the payload